### ISSUE TITLE
in count_sketch.py ,,,,,,,,,conved = tf.batch_ifft(tf.batch_fft(pc1) * tf.batch_fft(pc2))

### ISSUE THREAD

**Issue by haibin894609937** [therne/compact-bilinear-pooling-tf#5]
conved = tf.batch_ifft(tf.batch_fft(pc1) * tf.batch_fft(pc2))

in tensorflow r1.3

conved = tf.ifft(tf.fft(pc1) * tf.fft(pc2))
is it right？

**Comment by manupillai308** [comment]
Yes. I have ran the test script provided. For newer versions of tensorflow, 
```tf.ifft``` & ```tf.fft``` must be used.
